index.html

```diff
@@ -14,6 +14,57 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <img alt="Karma logo" class="logo-img" src="img/karma-logo.svg">
+        <nav>
+            <ul>
+                <li><a class="nav-head" href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
 
+    <main>
+        <section class="section1">
+            <p class="section1-phrase1">Introducing Karma</p>
+            <p class="section1-phrase2">Bring WiFi with you, everywhere you go.</p>
+            <button>Learn More</button>
+        </section>
+
+        <section class="section2">
+            <p class="section2-phrase1">Everyone needs a little Karma.</p>
+            <section class="section2-imgs-text">
```

Comment:
Sections generally want titles so this is probably not suitable to be a section a list of items could be better eg 
```
<ul>
   <li>  </li>
   <li>  </li>
</ul>
```